python/cugraph/structure/graph.py

```diff
@@ -113,7 +125,7 @@ def from_cudf_edgelist(self, input_df, source='source',
         source : str
             source argument is source column name
         destination : str
-            destination argument is destination column name.
+            target argument is destination column name.
```

Comment:
We should use `destination` consistently

Reply:
fixed